Chunk 1:	Change PREFIX and MANPREFIX variables
Chunk 2:	Append local CPPFLAGS, use _DEFALUT SOURCE
diff -ura sfeed.old/.git/index sfeed.new/.git/index
--- sfeed.old/Makefile	2020-03-25 13:29:05.092599907 +1100
+++ sfeed.new/Makefile	2020-03-25 13:27:09.328856315 +1100
@@ -4,16 +4,16 @@
 VERSION = 0.9.16
 
 # paths
-PREFIX = /usr/local
-MANPREFIX = ${PREFIX}/man
-DOCPREFIX = ${PREFIX}/share/doc/${NAME}
+PREFIX = /usr
+MANPREFIX = ${PREFIX}/share/man
+DOCPREFIX = ${PREFIX}/share/doc/sfeed-git
 
 RANLIB = ranlib
 
 # use system flags.
 SFEED_CFLAGS = ${CFLAGS}
 SFEED_LDFLAGS = ${LDFLAGS}
-SFEED_CPPFLAGS = -D_POSIX_C_SOURCE=200809L -D_XOPEN_SOURCE=700 -D_BSD_SOURCE
+SFEED_CPPFLAGS = ${CPPFLAGS} -D_POSIX_C_SOURCE=200809L -D_XOPEN_SOURCE=700 -D_DEFAULT_SOURCE
 
 BIN = \
 	sfeed\
